src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="input">
+          <label for="sname">Surname:</label>
+          <input
+          type="text"
+          id="sname"
+          name="sname"
+          required
+          autocomplete="off">
+        </div>
+        <div class="input">
```

Comment:
Better to separate large multiline siblings with empty lines.
```suggestion
        </div>
        
        <div class="input">
```